### PR TITLE
Refactor chatgptapi constructor

### PR DESCRIPTION
**Pull request by GoBenGoh**

### CODE REVIEW

File: LLM Testing/Code-Testing/src/main/java/ChatGPTAPI.java [hunk 1]
```diff
@@ -127,7 +97,8 @@ public void runTestIterations(String[] args, String repo, String commit, int com
             System.out.println(attempt);
 
             if (isStart){
-                resultAnalyzer = startTesting(app, "1", args, repo, attempt, CSVCreator, resultAnalyzer);
+                resultAnalyzer = startTesting("1", args, repo, attempt, CSVCreator, resultAnalyzer);
+                isStart = false;
                 if (!this.badResponse){
                     isStart = false;
                 }
```
Comment (cmrnnathan) [inline LLM Testing/Code-Testing/src/main/java/ChatGPTAPI.java]: Were you wanting to have isStart for both cases?

**Review by cmrnnathan — COMMENTED** [review]
Just wanted to check something

File: LLM Testing/Code-Testing/src/main/java/ChatGPTAPI.java (hunk 1 above)
Comment (GoBenGoh) [reply LLM Testing/Code-Testing/src/main/java/ChatGPTAPI.java]: Oops that was coz of a merge conflict. I'll fix it now.

**Review by cmrnnathan — APPROVED** [review]
Looks good

**Pull request merged**